### ISSUE TITLE
can not find data

### ISSUE THREAD

**Issue by nihao2333** [SangbumChoi/MobileHumanPose#40]
Download Human3.6M parsed data [[data](https://drive.google.com/drive/folders/1kgVH-GugrLoc9XyvP6nRoaFpw3TmM5xK?usp=sharing)]
Download MPII parsed data [[images](http://human-pose.mpi-inf.mpg.de/)][[annotations](https://drive.google.com/drive/folders/1MmQ2FRP0coxHGk0Ntj0JOGv9OxSNuCfK?usp=sharing)]
Download MuCo parsed and composited data [[data](https://drive.google.com/drive/folders/1yL2ey3aWHJnh8f_nhWP--IyC9krAPsQN?usp=sharing)]
Download MuPoTS parsed data [[images](http://gvv.mpi-inf.mpg.de/projects/SingleShotMultiPerson/)][[annotations](https://drive.google.com/drive/folders/1WmfQ8UEj6nuamMfAdkxmrNcsQTrTfKK_?usp=sharing)]
these links are 404.

**Comment by dubrovin-sudo** [comment]
Hi! I have the same problem

**Comment by nihao2333** [comment]
这是来自QQ邮箱的自动回复邮件。QQ邮箱已收到您的邮件，邮箱主人将在看到邮件之后尽快回复

**Comment by nihao2333** [comment]
Maybe you could use this project's data https://github.com/mks0601/3DMPPE_ROOTNET_RELEASE  
 they look the same

**Comment by SangbumChoi** [comment]
@nihao2333 @dubrovin-sudo Yes I did use the following project. You have to manually download official dataset from website if the link is deprecated.

**Comment by nihao2333** [comment]
这是来自QQ邮箱的自动回复邮件。QQ邮箱已收到您的邮件，邮箱主人将在看到邮件之后尽快回复
